addons/zir/functions/fnc_surrenderCheck.sqf

```diff
@@ -0,0 +1,49 @@
+#include "script_component.hpp"
+/*
+    Author:
+        Bridg
+
+    Description:
+        Checks if remaining units in a group should surrende
+        Uses an MPKilled EH that will check after a unit dies if
+        theres 2 unit or less
+    Parameter(s):
+        0: Unit <OBJECT>
+    Return Value:
+        Nothing
+*/
+params ["_unit"];
+
+_unit addMPEventHandler ["MPKilled", {
```

Comment:
MPKilled executes on all clients, recommend exiting if not local, or not on the server